react-toggle-switch-component/src/toggle-switch.jsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+
+export default class ToggleSwitch extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = { isClicked: false };
+    this.handleToggle = this.handleToggle.bind(this);
+  }
+
+  handleToggle() {
+    this.setState({ isClicked: !this.state.isClicked });
+  }
+
+  render() {
+    // console.log(this.state);
+    if (this.state.isClicked === false) {
+      return <div onClick={this.handleToggle} className='border off'>
+        <div className='switch-off'></div>
+      </div>;
+    } else if (this.state.isClicked === true) {
+      return <div onClick={this.handleToggle} className='border on'>
+        <div className='switch-on'></div>
+      </div>;
+    }
```

Comment:
This is okay, but can be done with a single `return` only a couple things change between each `return`.

```JSX
let status = 'off';

if (this.state.isClicked) {
  status = 'on';
}

return (
  <div onClick={this.handleToggle} className={`border ${status}`}>
    <div className={`switch-${status}`}></div>
  </div>
);
```